src/lib/openapi/spec/playground-feature-schema.ts

```diff
@@ -1,35 +1,110 @@
 import { FromSchema } from 'json-schema-to-ts';
+import { parametersSchema } from './parameters-schema';
 import { variantSchema } from './variant-schema';
 import { overrideSchema } from './override-schema';
+import {
+    playgroundStrategyEvaluation,
+    playgroundStrategySchema,
+} from './playground-strategy-schema';
+import { playgroundConstraintSchema } from './playground-constraint-schema';
+import { playgroundSegmentSchema } from './playground-segment-schema';
+
+export const unknownFeatureEvaluationResult = 'unevaluated' as const;
 
 export const playgroundFeatureSchema = {
     $id: '#/components/schemas/playgroundFeatureSchema',
     description:
         'A simplified feature toggle model intended for the Unleash playground.',
     type: 'object',
     additionalProperties: false,
-    required: ['name', 'projectId', 'isEnabled', 'variant', 'variants'],
+    required: [
+        'name',
+        'projectId',
+        'isEnabled',
+        'isEnabledInCurrentEnvironment',
+        'variant',
+        'variants',
+        'strategies',
+    ],
     properties: {
-        name: { type: 'string', example: 'my-feature' },
-        projectId: { type: 'string', example: 'my-project' },
-        isEnabled: { type: 'boolean', example: true },
+        name: {
+            type: 'string',
+            example: 'my-feature',
+            description: "The feature's name.",
+        },
+        projectId: {
+            type: 'string',
+            example: 'my-project',
+            description: 'The ID of the project that contains this feature.',
+        },
+        strategies: {
+            type: 'object',
+            additionalProperties: false,
+            required: ['result', 'data'],
+            properties: {
+                result: {
+                    description: `The cumulative results of all the feature's strategies. Can be \`true\`, \`false\`, or \`${playgroundStrategyEvaluation.unknownResult}\`. This property will only be \`${playgroundStrategyEvaluation.unknownResult}\` if one or more of the strategies can't be fully evaluated and the rest of the strategies all resolve to \`false\`.`,
```

Comment:
Could split these lines for readability:

```ts
result: {
    description: `
    	The cumulative results of all the feature's strategies. Can be \`true\`, \`false\`, or \`${playgroundStrategyEvaluation.unknownResult}\`. 
    	This property will only be \`${playgroundStrategyEvaluation.unknownResult}\` 
    	if one or more of the strategies can't be fully evaluated and the rest of the strategies all resolve to \`false\`.
    `,
},
```

Reply:
I like that suggestion!

Reply:
d1013c54b8b37910f8d0cd670a2427e2170239a5